azure-function/dotnet/Models/Entity.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GetBusData.Models;
+
+public sealed class Entity
+{
+    [JsonProperty("id")]
+    public string? Id { get; set; }
+
+    [JsonProperty("vehicle")]
```

Comment:
I don't think this `JsonProperty(...)` is necessary because the function app applies camelCasing by default.